### ISSUE TITLE
Routing the App (let's keep building on this)

### ISSUE THREAD

**Issue by jandony** [jandony/react-workout#11]
Muscle Group Cards (on Exercise Finder tab):
1. Route each card to it's own "results page" of workouts that fall under each category (add route in arrays)
2. Find a way to categorize each exercise (maybe create another array for just categories and filter?)

Your Workouts
1. By clicking a Workout (playlist), you will see the individual list of exercises. Each workout should be it's own route

**Comment by AvatarGreggles** [comment]
@jandony So, for the first part, for Muscle Group Cards, do you envision say, a user clicks, the Neck Card, that it will replace the excercise finder with a list of the Neck related excercies and the URL would be like http://xxxxx.com/neck-workouts

**Comment by AvatarGreggles** [comment]
I also think we can condense the exercise finder route into a sidebar component which allows the user to easily navigate the exercises even when the results are shown? If you need me to draw up a sketch of what I mean let me know :)

**Comment by jandony** [comment]
Yup! That's the only way I know how to route, but it doesn't seem very dynamic though huh?

**Comment by AvatarGreggles** [comment]
@jandony See my comment above, this way would make it more dynamic I feel.

**Comment by jandony** [comment]
Ohhh I see. Would we replace the current muscle group cards with a sidebar? Or would we have both?

**Comment by AvatarGreggles** [comment]
If we have a fixed sidebar that is present on all pages (its the main focus of the app is to navigate the exercises) then we can allow for the user to search and add exercises whenever they want.

**Comment by AvatarGreggles** [comment]
The cards can be condensed down into more like a nested listed that expanded out with exercises instead of updatinf the route and navigating to different pages and away from the current context.

**Comment by AvatarGreggles** [comment]
I could whip up the component anyway AND keep the current exercise finder so we can compare and use both if we want to.

**Comment by jandony** [comment]
Sounds good to me! :) Let's try it!
